e2e/client/playwright/templates.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('creating new template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-header')).getByRole('button', {name: 'Add new'}).click();
+
+    await page.locator(s('template-edit-view')).getByPlaceholder('template name').fill('Template 1');
+    await page.locator(s('template-edit-view', 'select-content-profile')).selectOption({label: 'Story'});
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template--template 1'))).toBeVisible();
+});
+
+test('editing template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template--story', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('template-edit-view')).getByPlaceholder('template name').fill('story 1.1');
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template--story 1.1'))).toBeVisible();
+});
+
+test('removing template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template--story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Remove'}).click();
+    await page.locator(s('modal-confirm')).getByRole('button', {name: 'Ok'}).click();
+    await expect(page.locator(s('template-content', 'content-template--story 2'))).not.toBeVisible();
+});
+
+test('Assign template to a desk', async ({page}) => {
+    const monitoring = new Monitoring(page);
+
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Finances');
+
+    await page.locator(s('content-create')).click();
+    await page.locator(s('content-create-dropdown')).getByRole('button', {name: 'More Templates...'}).click();
+    await expect(page.locator(s('content-create-dropdown')).getByRole('button', {name: 'Story 2'})).not.toBeVisible();
+
+    // assign template to the desk
+    await page.goto('/#/settings/templates');
+    await page.locator(s('template-content', 'content-template--story 2', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
+    await page.locator(s('template-edit-view', 'desks', 'desk--Finances')).first().click();
```

Comment:
It would be ideal to use the `.check()` method for checkboxes. Example: `expect(page.getByLabel('abc')).check();`

This doesn't look like a checkbox, but we could use role="checkbox". If it doesn't work out easily - at least add a comment that you expect that checkbox is off and by clicking it - you are toggling it on